ReeldIn/settings.py

```diff
@@ -134,6 +134,8 @@
 
 AUTH_USER_MODEL = "accounts.User"
 
+LOGIN_URL = "login/"
```

Comment:
For the login_required decorator. I found online that by default it will redirect to the login route specified in settings.py. I added this so I wouldn't have to manually specify the route in each view. I'm not 100% sure that this is the correct path though